service_capacity_modeling/models/org/netflix/kafka.py

```diff
@@ -367,12 +412,12 @@ def default_desires(user_desires, extra_model_arguments: Dict[str, Any]):
                 # This = 1000 / 14 = 71 millisecond / MiB
                 # To read 1 MiB costs 71 millisecond of CPU time
                 estimated_mean_read_latency_ms=Interval(
-                    low=50, mid=70, high=100, confidence=0.98
+                    low=30, mid=40, high=100, confidence=0.98
```

Comment:
thinking we can probably lower this to `low=20, mid=30, high=75` based on empirical data for both reads and writes.